core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java

```diff
@@ -230,6 +237,13 @@ protected final long getNextDelay(ClientRequestContext ctx, Backoff backoff, lon
         return nextDelay;
     }
 
+    /**
+     * Returns the total attempts number of the current request.
```

Comment:
`Returns the total number of attempts of the request represented by the specified {@link ClientRequestContext}.` ?